github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/context/GitHubPRTriggerDslContext.java

```diff
@@ -33,6 +35,14 @@ public void setPreStatus() {
         setPreStatus = true;
     }
 
+    public void setCancelQueued() {
```

Comment:
`setPreStatus` is not just setter. It means "set pre status on github". When in `setCancelQueued` `set` is just setter. You can't say "set cancel queued jobs" (its ridiculous)